ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/productdetails.html

```diff
@@ -0,0 +1,31 @@
+<div class ='product-details-cmp'>
+<div class ='product-details-cmp__img aem-Grid aem-Grid--9 aem-Grid--default--12'>
+   <img class ='product-details-cmp--product-img' src="${properties.fileReference}" alt="image"/>
+</div>
+<div class ='product-details-cmp__product-details aem-Grid aem-Grid--3 aem-Grid--default--12'>
+   <b class ='product-details-cmp__prduct-name'>${properties.productName} </b>
+   <div>
+      <span class="product-details-cmp__prduct-price-sign">$</span>
+      <p class ='product-details-cmp__prduct-price'></p>
+   </div>
+   <div class ='product-details-cmp__prduct--description'>${properties.description @ context='html'}</div>
+   <sly data-sly-resource="${'accordion' @ resourceType='hclecomm/components/content/accordion', decoration=true}"> </sly>
```

Comment:
@Rajamouli-hcl   Why Return Policy accordion is not loading.?

Reply:
please check its working